src/MiniCover/Commands/Options/CoverageHitsFileOption.cs

```diff
@@ -0,0 +1,41 @@
+﻿using Microsoft.Extensions.CommandLineUtils;
+using System;
+
+namespace MiniCover.Commands.Options
+{
+    public class CoverageHitsFileOption : IMiniCoverOption<string>
+    {
+        private const string DefaultValue = "coverage-hits.txt";
+        private bool _invalidated;
+        private CommandOption _option;
+        private string _value;
+        public string Description => $"Hits file name pattern [default: {DefaultValue}]";
+        public string OptionTemplate => "--hits-file";
+        public CommandOptionType Type => CommandOptionType.SingleValue;
+
+        public string Value
+        {
+            get
+            {
+                if (_invalidated) return _value;
+                throw new MemberAccessException("Option should be invalidate before Value access");
+            }
+            set => _value = value;
+        }
+
+        public void Initialize(CommandLineApplication commandContext)
+        {
+            _option = commandContext.Option(OptionTemplate, Description, Type);
+        }
+
+        public void Invalidate()
+        {
+            var proposalValue = _option.Value();
+            if (string.IsNullOrEmpty(proposalValue))
```

Comment:
how about using `string.IsNullOrWhiteSpace` instead?

Reply:
@rohitjha CommandOption is doing that part but for clean code and keep an independent solution it is a good idea. (updated)